ui/src/components/InputLabel/index.tsx

```diff
@@ -27,26 +84,42 @@ export default function InputLabel({
             {fields.map(({ key, name }, index) => (
               <Row gutter={8} style={{ marginBottom: 8 }} key={key}>
                 <Col span={11}>
-                  <Form.Item name={[name, labelFormName]} noStyle>
+                  <LableFormItem
+                    name={[name, labelFormName]}
+                    fieldIdx={name}
+                    dependName={[name, valueFormName]}
```

Comment:
Is this `valueFormName` confused with following form item? Why use `valueFormName` with `label` and `labelFormName` with `value`?

Reply:
`valueFormName` means the form name of the value input box in the Label component.

Reply:
The `label name` and `label value` verification are dependent on each other. When one of them is empty, the other can also be empty; when one of them is not empty, the other is not empty either.